packages/types/definitions.ts

```diff
@@ -253,10 +253,9 @@ export type Scenario = {
        * @title Scenario flow
        */
       flow: Array<
-        | HttpFlow
         | WebSocketFlow
         | ({
-            loop: Array<HttpFlow | WebSocketFlow>;
```

Comment:
TIL The `websocket` engine doesn't support the HTTP flow but the `socketio` engine [does](https://www.artillery.io/docs/reference/engines/socketio#mixing-in-http-actions). Thanks @bernardobridge for pointing this out.